APP/RunRules/Vectis/NBody.xml

```diff
@@ -29,6 +29,7 @@
             <classpath>
                 <pathelement path="${env.ANT_MAXELER_INTERNAL_MAXIDE_JARS}"/>
                 <pathelement path="../RunRules/Vectis/dist/bin"/>
+                <pathelement path="/network-raid/opt/maxq/maxq-ctl.jar"/>
```

Comment:
@djovanovic-maxeler: Will this make problems for the automatic AppGallery apps testing?

There are more hardcoded paths, please also suggest changes for those if needed.